lm_eval/models/huggingface.py

```diff
@@ -13,6 +13,7 @@
     InitProcessGroupKwargs,
     find_executable_batch_size,
 )
+from accelerate.utils import is_npu_available
```

Comment:
As of what `accelerate` version is NPU available? this will likely error for earlier accelerate versions?

Reply:
`accelerate` adds support for Ascend NPU starting from version [0.21.0](https://github.com/huggingface/accelerate/releases/tag/v0.21.0)